genapi/src/formula.rs

```diff
@@ -514,7 +500,35 @@ impl<'a> Parser<'a> {
             Expr::Float(f)
         } else {
             let s = self.next_ident().unwrap();
-            Expr::Ident(s)
+            if self.eat(&Token::LParen) {
+                let op = Some(match s.as_str() {
+                    "NEG" => UnOpKind::Neg,
+                    "SIN" => UnOpKind::Sin,
+                    "COS" => UnOpKind::Cos,
+                    "TAN" => UnOpKind::Tan,
+                    "ASIN" => UnOpKind::Asin,
+                    "ACOS" => UnOpKind::Acos,
+                    "ATAN" => UnOpKind::Atan,
+                    "ABS" => UnOpKind::Abs,
+                    "EXP" => UnOpKind::Exp,
+                    "LN" => UnOpKind::Ln,
+                    "LG" => UnOpKind::Lg,
+                    "SQRT" => UnOpKind::Sqrt,
+                    "TRUNC" => UnOpKind::Trunc,
+                    "FLOOR" => UnOpKind::Floor,
+                    "CEIL" => UnOpKind::Ceil,
+                    "ROUND" => UnOpKind::Round,
+                    _ => return Expr::Ident(s),
```

Comment:
```suggestion
                    other =>  _ => panic!("{} is not a keyword for function call", other),
```
Currently,  the parser of `cameleon-genapi` is designed to panic if input XML doesn't meet GenApi specification, though it may return `Err` in the future.

Reply:
Adressed in f419ab06344ba59663868abac1c3ad53c3879ef2